gallery_events.html

```diff
@@ -52,7 +53,7 @@
 
       <div class="col-6 col-xl-2" data-aos="fade-down">
         <!-- <h1 class="mb-0"><a href="index.html" class="text-black h2 mb-0">SPEC</a></h1>-->
-        <a href="index.html"><img src="gallery\bg-img\spec_logo.png" class="logo" /></a>
+        <a href="index.html"><img src="imgs\headicon.ico" class="logo" /></a>
```

Comment:
@baamaboi, is this working fine with .ico file?

Reply:
yes